### PR TITLE
[LPS-138717,LPS-139643,LPS-139645,LPS-139789] Various Remote Apps Adjustments

### PR DESCRIPTION
**Pull request by jbalsas**
This is https://github.com/liferay-frontend/liferay-portal/pull/1488 minus [LPS-139629](https://issues.liferay.com/browse/LPS-139629) plus [LPS-139789](https://issues.liferay.com/browse/LPS-139789), so:
- [Custom elements add empty CSS resources that fail fetching](https://issues.liferay.com/browse/LPS-138717)
- [Remove unnecessary scroll from DatasetDisplay](https://issues.liferay.com/browse/LPS-139643)	
- [The remote-app-client SDK is being rendered unnecessarily](https://issues.liferay.com/browse/LPS-139645)
- [Can't add multiple URLs to Remote App Custom Element on first try](https://issues.liferay.com/browse/LPS-139789)

/cc @izaera @john-co, please see [LPS-139789](https://issues.liferay.com/browse/LPS-139789). This was caught manually by @kevenleone. Could we maybe add some automation for this?

### CODE REVIEW

File: modules/apps/remote-app/remote-app-web/src/main/java/com/liferay/remote/app/web/internal/portlet/action/EditRemoteAppEntryMVCActionCommand.java [hunk 1]
```diff
@@ -96,11 +96,16 @@ private void _add(ActionRequest actionRequest) throws PortalException {
 		String type = ParamUtil.getString(actionRequest, "type");
 
 		if (type.equals(RemoteAppConstants.TYPE_CUSTOM_ELEMENT)) {
+			String[] customElementCSSURLs = ParamUtil.getStringValues(
```
Comment (nhpatt) [inline modules/apps/remote-app/remote-app-web/src/main/java/com/liferay/remote/app/web/internal/portlet/action/EditRemoteAppEntryMVCActionCommand.java:99]: Brian is in everything-should-be-inlined mode so maybe he inlines this, but I prefer with a explicit variable and readable method calls and is not consistent with it, so I'll leave it this way, just FYI.

**Review by nhpatt — APPROVED** [review]

**Comment by nhpatt** [comment]
LGTM :) We've reviewed 3 commits in the past and I want to use the rare green in the CI, so forwarding.

**Comment by nhpatt** [comment]
ci:forward

File: modules/apps/remote-app/remote-app-web/src/main/java/com/liferay/remote/app/web/internal/portlet/action/EditRemoteAppEntryMVCActionCommand.java (hunk 1 above)
Comment (jbalsas) [reply modules/apps/remote-app/remote-app-web/src/main/java/com/liferay/remote/app/web/internal/portlet/action/EditRemoteAppEntryMVCActionCommand.java:99]: Yeah, hard to know... I just copied this from the `_update` method down below in this same file, so either this is kept like this, or both should be updated 🤷

**Comment by john-co** [comment]
Thanks @jbalsas, will include in next round testing and automation: https://issues.liferay.com/browse/LPS-139804

**Pull request closed without merging**